packages/@o3r/rules-engine/package.json

```diff
@@ -111,10 +114,12 @@
     "@nx/jest": "~18.2.0",
     "@nx/js": "~18.2.0",
     "@o3r/build-helpers": "workspace:^",
+    "@o3r/configuration": "workspace:^",
```

Comment:
The rules engine should not have dependency to this packages

Reply:
the new added deps are used in the integration test. They are added as dev-deps only